Specs/DateTime/German/DateExtractor.json

```diff
@@ -28,7 +28,7 @@
     "Context": {
       "ReferenceDateTime": "2016-01-01T00:00:00"
     },
-    "NotSupported": "javascript, python",
+    "NotSupported": "javascript, python, java",
```

Comment:
If you're adding German support, why are these not supported? Will support still come in a follow up PR?
Please add a comment in the description summarizing why these few cases are not yet supported to the best of your analysis of the reasons.